src/main/java/com/amazonaws/encryptionsdk/kms/KmsMasterKeyProviderBuilder.java

```diff
@@ -13,11 +16,10 @@
 import java.util.List;
 
 /**
- * Used to build a {@link KmsMasterKeyProvider} using {@link AWSKMSClientBuilder}, which will use provider chains to get
- * defaults if the properties are not explicitly set.
+ * Used to build a {@link KmsMasterKeyProvider} which will use the defaults found in the relevant provider chains.
  *
- * Default usage will use all the defaults found by AWSKMSClientBuilder. If a region is not found in the configuration
- * chain, then {@link Regions#DEFAULT_REGION} is used.
+ * If a region is not defined and no region can be found in the region provider chain, then
+ * {@link Regions#DEFAULT_REGION} is used.
```

Comment:
I am using `Regions#DEFAULT_REGION` if no region can be found, because that is what the constructors are currently doing. However, the Java SDK now throws an exception if a region cannot be found in the region provider chain. Should the builder follow the Java SDK builders? More on why: 

> If you create a client through the constructor, it initializes the client with a default endpoint (typically, us-east-1 or a global endpoint). For this reason, many customers don't bother setting a region because it just works with the default endpoint. When these customers attempt to deploy their application to another region, a lot of code has to be changed to configure the client with the correct region based on the environment.

https://java.awsblog.com/post/Tx3QE4T4596T5G4/Fluent-Client-Builders